### PR TITLE
LOC CHECKIN | testfx - master | 20171129

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf [hunk 1]
```diff
@@ -5,306 +5,306 @@
...
       </trans-unit>
       <trans-unit id="UTA_CleanupStackTrace">
         <source>TestCleanup Stack Trace</source>
-        <target state="new">TestCleanup Stack Trace</target>
-        <note></note>
+        <target state="translated">Trasování zásobníku čištění testu</target>
+        <note />
       </trans-unit>
       <trans-unit id="DiscoveryWarning">
         <source>[MSTest][Discovery][{0}] {1}</source>
```
Comment (singhsarab) [inline src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: @jayaranigarg Any idea why is this in resx if we don't expect it to get localized ?

Comment (jayaranigarg) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: @v-zbsail : Can you check why this string is not translated, even though it is marked as translated?

Comment (janbaltus) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: Do you refer to "TestCleanup Stack Trace"?

Comment (jayaranigarg) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: @janbaltus : I meant "[MSTest][Discovery][{0}] {1}" string..

Comment (janbaltus) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: Is this string to be translated?
Can you please add more context?

Comment (jayaranigarg) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: Yes, I believe it should have been translated like the rest of the strings.
We plan to remove [MSTest] from this string though as we do not want MSTest to be translated but would want [Discovery] to be translated. However, the change of removing [MSTest] will be done in a separate PR.

Comment (janbaltus) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: Ince you update the xlf we will fix this in next round of localization.
Please remove also Discovery or add note for that string explaining this is localizable.
Any string in [] {} look like placeholder from our point of view

Comment (jayaranigarg) [reply src/Adapter/MSTest.CoreAdapter/Resources/xlf/Resource.cs.xlf:305]: Thanks @janbaltus . Will do those changes later. Meanwhile I will push this PR in.

**Comment by jayaranigarg** [comment]
@dotnet-bot test Windows / Debug Build  please

**Comment by jayaranigarg** [comment]
@dotnet-bot test this please

**Pull request merged**